shared/src/main/java/com/couchbase/lite/ArrayExpression.java

```diff
@@ -41,8 +43,13 @@ private ArrayExpression() {
      *
      * @param variable The variable expression.
      * @return An In object
+     * @throws IllegalArgumentException when variable is null.
```

Comment:
Do we have this in other places such as Database class? If not, please remove as it’s not necessary.

Same for other places.